### PR TITLE
dashboard styling updates

### PR DESCRIPTION
**Pull request by ckiss**
Contains: 
- dashboard styling update to match designs
- county selector dropdown on dashboard
- styled the two cards: contact alert and tracing available 
- updated icons from Antoine

Missing: 
- correct data for the two charts on dashboard (do we have that already?!)
- take action when county is changed on dashboard

### CODE REVIEW

File: src/assets/lang/en.json [hunk 1]
```diff
@@ -388,8 +394,7 @@
     "description": "Please check the COVID Green app now."
   },
   "closeContactWarn": {
-    "title": "Close Contact Alert",
-    "notice": "The app has detected that you have been in close contact with someone who has tested positive for COVID-19."
+    "notice": "You've had {{contactsNumber}} COVID Alert in the last 14 days"
```
Comment (ShaunBaker) [inline src/assets/lang/en.json]: If more than 1 alert - should it not be "Alerts"?

File: src/components/molecules/county-dropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import {SelectorDropdown} from 'components/atoms/dropdown';
+import {counties} from 'assets/counties';
+
+interface CountyOption {
+  label: string;
+  value: string;
+}
+
+const countyOptions: CountyOption[] = [];
+const map = new Map();
```
Comment (ShaunBaker) [inline src/components/molecules/county-dropdown.tsx]: What's the purpose of this map?

**Review by ShaunBaker — COMMENTED** [review]
A couple minor comments/queries.

A suggestion would be to run the new/updates SVG through SVGOMG

File: src/components/molecules/county-dropdown.tsx (hunk 2 above)
Comment (ckiss) [reply src/components/molecules/county-dropdown.tsx]: some leftover, is not needed here.

**Review by ShaunBaker — APPROVED** [review]

**Pull request merged**